xml/dns_versionbind.xml

```diff
@@ -0,0 +1,685 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  This fingerprint file matches the text string response from a DNS
+  version.bind request.
+
+  For example, the string 'dnsmasq-2.76-1-ubnt2' emitted by the command below:
+
+  $ nslookup -type=txt -class=chaos VERSION.BIND <dns_server> | grep VERSION.BIND | cut -d\" -f2
+  dnsmasq-2.76-1-ubnt2
+
+-->
+<fingerprints matches="dns_versionbind" protocol="dns" database_type="service" preference="0.750">
```

Comment:
Ah, I hadn't made the connection between the two different naming problems and agree that is problematic.

Part of the complication here is that there is the implication that the name of the files matter and perhaps aside from some nuanced ordering issues and the fact that external consumers of recog XML content might depend on the names of the files, the names of the files don't matter but `matches` does.

This is complicated by the fact that before `matches`, the name of th file was used to imply what the file contained and how it was to be used.

For now, do whatever makes the most sense for you.  Longer term we can find a better solution for this.